app/core/migrations/0005_sponsorpartner.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 4.0.7 on 2022-09-21 23:36
+
+from django.db import migrations, models
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0004_recurringevent'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='SponsorPartner',
+            fields=[
+                ('uuid', models.UUIDField(default=uuid.uuid4, editable=False, primary_key=True, serialize=False, unique=True)),
+                ('created_at', models.DateTimeField(auto_now_add=True, verbose_name='Created at')),
+                ('updated_at', models.DateTimeField(auto_now=True, verbose_name='Updated at')),
+                ('partner_name', models.CharField(max_length=255)),
+                ('partner_logo', models.URLField(blank=True)),
```

Comment:
As I suggested in the User model, I  think you probably will want to convert this to an ImageField so you could upload the logos rather than relying on the partner logos to remain at a specific url on their site. But I am not going to hold up this PR for that change. You can create a new one if you agree with this suggestion.